src/nvector/pthreads/nvector_pthreads.c

```diff
@@ -903,18 +935,26 @@ void N_VScale_Pthreads(sunrealtype c, N_Vector x, N_Vector z)
   if (z == x)
   { /* BLAS usage: scale x <- cx */
     VScaleBy_Pthreads(c, x);
+    SUNCheckLastErrVoid();
```

Comment:
I thought we weren't checking internal routines that do not return a `SUNErrCode`?

Reply:
We are not, I think I added this because of a suggestion and I did not notice the context. Ill remove it.